.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,32 @@
+Description of Issue/Question
+
+
+### Did you follow the steps from https://github.com/napalm-automation/napalm#faq
+[ ] Yes
```

Comment:
Can do:

```
- [ ] Yes
- [ ] No
```

to look like:

- [x] Yes
- [ ] No

When checked. That was missed for other drivers, but is not that important anyway.